src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/ClusterAwareWriterFailoverHandler.java

```diff
@@ -67,6 +69,7 @@
   protected int maxFailoverTimeoutMs = 60000; // 60 sec
```

Comment:
Similarly to ConnectionAttemptTask in ClusterAwareReaderFailover, should we make the two static classes in this class non-static so that we don't have to pass in so many parameters to the constructor?

Reply:
Yes, it's a good idea.